addons/l10n_lt/data/account.account.template.csv

```diff
@@ -0,0 +1,184 @@
+"account_account_template_20210","Cost of Executive Works",20210,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_2","False"
+"account_account_template_2030","Cost of Production",2030,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_3","False"
+"account_account_template_2035","Production in Transit",2035,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_3","False"
+"account_account_template_2036","Production at Third Parties",2036,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_3","False"
+"account_account_template_2039","Value Loss of Production (-)",2039,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_3","False"
+"account_account_template_2040","Cost of Purchased Goods for Resale",2040,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_4","False"
+"account_account_template_2045","Purchased Goods for Resale in Transit",2045,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_4","False"
+"account_account_template_2046","Purchased Goods for Resale at Third Parties",2046,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_4","False"
+"account_account_template_2049","Value Loss of Purchased Goods for Resale (-)",2049,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_4","False"
+"account_account_template_2060","Cost of Tangible Assets",2060,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_6","False"
+"account_account_template_2080","Prepayments to Suppliers",2080,"account.data_account_type_prepayments","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_7","False"
+"account_account_template_2084","Deposit",2084,"account.data_account_type_prepayments","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_7","False"
+"account_account_template_2410","Value of Debts from Customers",2410,"account.data_account_type_receivable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_1","True"
+"account_account_template_24400","Value of Loans Granted",24400,"account.data_account_type_receivable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_4","True"
+"account_account_template_2441","Receivable Value-Added Tax",2441,"account.data_account_type_receivable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_4","True"
```

Comment:
```suggestion
"account_account_template_2441","Receivable Value-Added Tax",2441,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_2_4","True"
```

Reply:
Those accounts are kind of payable/receivable in taxes sense, but those are not real payable/receivable. So I agree, it should not be payable/receivable. We only moved those here, when we were merging account types and were looking for some alternative. 

Then `reconcile` option should also be set to False (as it was originally intended), because accountant said it is not really needed to be reconcilable.